portfolio/src/main/webapp/styles/comments.css

```diff
@@ -1,17 +1,138 @@
 body {
-  background-color: white;
-  padding: 5%; 
+  background-color: white; 
+  font-family: 'Raleway', sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row; 
+  justify-content: center;
+  align-items: center; 
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+  background-color: black;
+  color: white;
+  text-align: center;
```

Comment:
When the entries get somewhat long like this, it's better to have them alphabetical so it's easier to find each style entry.

Reply:
Got it